client/src/views/AboutUsView.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class='about-us'>
+    <div class='text'>
+      <div class='h1-header-xxl'>{{ messages.HEADER }}</div>
+      <div class='h1-header'>
+        {{ messages.BODY_1 }}
+        <p></p>
+        {{ messages.BODY_2 }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang='ts'>
+import { defineComponent } from 'vue';
+import { AboutUs as messages } from '@/assets/messages/about_us';
+
+/**
+ * About Us page component.
+ */
+export default defineComponent({
+  name: 'AboutUsView',
+  data() {
+    return {
+      messages,
+    };
+  },
+});
+</script>
+
+<style scoped lang='scss'>
+@import '../assets/styles/global.scss';
+@import '@blueconduit/copper/scss/01_settings/design-tokens';
+
+.about-us {
+  height: 100%;
+  background-color: $light-blue-800;
+  background-image: url('~@/assets/media/about-us-image.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  color: $white;
+}
+
+.h1-header {
+  width: 169 * $spacing-xs;
```

Comment:
nit: multiplying by a factor here seems to defeat the purpose of using a common value. Would [`max-width`](https://github.com/BlueConduit/copper/blob/main/packages/copper/scss/01_settings/_design-tokens.scss#L55) work instead?

Reply:
Changed to max-width and just a px value. Left a TODO to add a var for these values. Thanks!